web-console/src/views/services-view/services-view.tsx

```diff
@@ -438,14 +437,15 @@ ORDER BY "rank" DESC, "service" DESC`;
             },
             Aggregated: row => {
               switch (row.row._pivotVal) {
-                case 'historical':
+                case 'historical': {
```

Comment:
Been working with ECMAScript languages since 2008 and did not know that you can have blocks in `switch` statements.